R/infrastructure.R

```diff
@@ -291,6 +291,27 @@ use_package_doc <- function(pkg = ".") {
   writeLines(out, file.path(pkg$path, path))
 }
 
+# help function for use_package
+# takes the same arguments as use_package
+build_package_txt <- function(package, version = NULL,
+                              compare =  c(">=", ">", "==", "<=", "<")) {
```

Comment:
Like I mentioned in the PR to @jimhester : it costs very little to keep it in. However, I can envision a case where someone is developing a package and wants to use `>=` and `<=` . Or, if they're writing production code that'll turn into a microservice in a docker container, `==` might also make sense.